nemo-keyboard/org/nemomobile/KeyboardLandscape.qml

```diff
@@ -153,16 +159,6 @@ Column {
             captionShifted: "."
             sizeType: "keyboard-key-120x46.png"
         }
-
-        EnterKey {
-            width: keyArea.width / 8
-            height: keyHeight
-            topPadding: keyArea.topPadding
-            leftPadding: keyArea.leftPadding
-            rightPadding: keyArea.rightPadding
-            bottomPadding: keyArea.bottomPadding
-            landscape: true
-        }
```

Comment:
It can be tested by changing, for example [TextInputPage](https://github.com/nemomobile/qtquickcontrols-nemo/blob/master/examples/touch/content/TextInputPage.qml), allowedOrientations to 
`allowedOrientations: Qt.LandscapeOrientation`

Reply:
yea....plz

Reply:
Enter key is not visible while in landscape
![screenshot_20180726_001](https://user-images.githubusercontent.com/11635400/43269417-7bd978b6-90fb-11e8-92a2-e15aad7fab90.png)
(and is broken because KLayouts not defined)^